setup.cfg

```diff
@@ -24,6 +24,7 @@ install_requires =
     parsimonious
     requests
     setuptools
+    singlestore-vectorstore>=0.1.1
```

Comment:
I noticed that this package require Python 3.9, whereas this SDK starts at 3.8. I know that 3.8 is unsupported, but we may get complaints from some customers that they can't install this update because of the 3.9 dependency because they are stuck at 3.8. I'd like to retire 3.8, so we can try it and see if anyone complains, but I thought I'd give you a heads-up.